jobs/generation/MetaGenerator.groovy

```diff
@@ -227,11 +227,13 @@ repos.each { repoInfo ->
         return;
     }
 
-    // Make the folders
+    // Make the folders. Save the root folder for overview generation
     def generatorFolder = ''
+    def projectFolder = ''
     for (folderElement in repoInfo.folders) {
         if (generatorFolder == '') {
             generatorFolder = folderElement
+            projectFolder = folderElement
```

Comment:
When explicit folders are specified, it should generate this view at the root folder, which if I understand how this script works, should still be the project name. As far as I can tell, the explicit folders are subfolders of the project name, correct? If that's true, this should still generate the one view, which will recurse into all folders and show everything that doesn't end in "_prtest".